### PR TITLE
:grin: new layout applied

### PR DESCRIPTION
**Pull request by ajthomascouk**

### CODE REVIEW

File: error-503.html [hunk 1]
```diff
@@ -7,125 +7,159 @@
...
+      <div class="notification-banner notification-banner--js-hidden" id="global-cookies-banner">
+        <p class="notification-banner--inner">
+        NHS.UK uses cookies to make the site simpler.
+        <a href="https://beta.nhs.uk/help/cookies">Find out more about cookies</a>
+        </p>
+      </div>
+
+      <div class="notification-banner notification-banner--hiviz js-feedback-banner">
+    <p class="notification-banner--inner">
+      This is new. <a class="back-to-choices" href="https://www.nhs.uk/Service-Search/Pharmacy/LocationSearch/10?nobeta=true">Find a pharmacy using our old service.</a>
```
Comment (st3v3nhunt) [inline error-503.html]: I don't think the pharmacy finder specific messaging and navigation should be included in the site wide error page.

File: error-503.html [hunk 2]
```diff
@@ -7,125 +7,159 @@
...
+          <li><a href="https://www.nhs.uk/Personalisation/Registration.aspx">Create an account</a></li>
+        </ul>
       </div>
-    </header>
+    </nav>
+    <!-- endnoindex -->
+  </header>
+
+
+      <main id="content" role="main" class="util-nofocus">
```
Comment (st3v3nhunt) [inline error-503.html]: I don't think the pharmacy finder specific messaging and navigation should be included in the site wide error page.

File: error-503.html [hunk 3]
```diff
@@ -7,125 +7,159 @@
...
+                https://assets.nhs.uk/images/nhs-logo.png 1x,
+                https://assets.nhs.uk/images/nhs-logo@2x.png 2x,
+                https://assets.nhs.uk/images/nhs-logo@3x.png 3x,"
+            />
+          </a>
+        </li>
+        <li><a href="http://www.nhs.uk/Pages/nhs-sites.aspx">NHS sites</a></li>
+        <li><a href="http://www.nhs.uk/Pages/about-us.aspx">About us</a></li>
+        <li><a href="http://www.nhs.uk/Pages/contact-us.aspx">Contact us</a></li>
+      </ul>
```
Comment (st3v3nhunt) [inline error-503.html:107]: Looks like a double `</ul>`

File: error-503.html [hunk 4]
```diff
@@ -7,125 +7,159 @@
...
+        <li>&copy; Crown Copyright</li>
+      </ul>
+    </div>
+  </div>
+
     </footer>
+    <script
+      src="https://code.jquery.com/jquery-1.12.4.min.js"
+      integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ="
+      cro
```
Comment (st3v3nhunt) [inline error-503.html]: I don't think the type attribute is required https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-type

File: error-503.html [hunk 5]
```diff
@@ -7,125 +7,159 @@
...
+      <header class="site-header">
+    <div class="site-header__top">
+      <div class="site-header__brand">
+        <!-- noindex -->
+        <a href="/">
+          <img
+            src="https://assets.nhs.uk/images/choices-logo.png" srcset="
+                https://assets.nhs.uk/images/choices-logo.png 1x,
+                https://assets.nhs.uk/images/choices-logo@2x.png 2x,
+                https://assets.nhs.uk/images/choices-logo@3x.png 3x,"
```
Comment (st3v3nhunt) [inline error-503.html]: The comma at the end of can be removed here as it is thought of as an empty element.

File: error-503.html [hunk 6]
```diff
@@ -7,125 +7,159 @@
...
+
+
+
+
+      <header class="site-header">
+    <div class="site-header__top">
+      <div class="site-header__brand">
+        <!-- noindex -->
+        <a href="/">
+          <img
```
Comment (st3v3nhunt) [inline error-503.html:35]: Can this img have an alt tag?

File: error-503.html (hunk 2 above)
Comment (st3v3nhunt) [inline error-503.html]: Is the role attribute needed here?
screen shot 2017-12-04 at 15 49 15

File: error-503.html [hunk 7]
```diff
@@ -7,125 +7,159 @@
...
-            <li>
-              <a href="https://beta.nhs.uk/help/clinical-assurance">Clinical assurance</a>
-            </li>
-            <li>
-              <a href="https://beta.nhs.uk/help/cookies">Cookies</a>
-            </li>
-          </ul>
-        </div>
-      </div>
+    <footer role="contentinfo" class="footer">
```
Comment (st3v3nhunt) [inline error-503.html]: Is the role attribute needed here?
screen shot 2017-12-04 at 15 49 21

File: error-503.html [hunk 8]
```diff
@@ -7,125 +7,159 @@
...
+      <div class="footer__main-links">
+    <div class="footer__content">
+      <ul>
+        <li>
+          <a href="/">
+            <img class="footer__logo" src="https://assets.nhs.uk/images/nhs-logo.png" srcset="
```
Comment (st3v3nhunt) [inline error-503.html:96]: Should this img tag have an alt attribute?

File: error-503.html [hunk 9]
```diff
@@ -7,125 +7,159 @@
...
...r-bottom:5px solid transparent;border-right:6px solid #0b0c0c;border-top:5px solid transparent;content:"";display:block;height:0;left:0;margin-top:-6px;position:absolute;top:50%;width:0}.link-back:link,.link-back:visited,.link-back:hover,.link-back:active{color:#000000}.error{margin-right:15px;border-left:4px solid #b10e1e;padding-left:10px}.error>.form-control{border:4px solid #b10e1e}@media (min-width: 641px){.error{border-left:5px solid #b10e1e;padding-left:15px}}.error-summary{border:4px solid #338ed1;margin-bottom:45px;padding:15px 10px;background-color:#fff}@media (min-width: 641px){.error-summary{border:5px solid #338ed1;margin-top:0;margin-bottom:45px;padding:20px 15px 15px}}.error-summary:focus{outline:3px solid #ffbf47}.error-summary-heading{margin-top:0}.error-summary-list{color:#000;list-style-type:none;margin-top:0;padding:0}@media (min-width: 641px){.error-summary-list li{margin-bottom:5px}}.error-summary-list a{color:#b10e1e;font-weight:bold;text-decoration:underline}.error-summary p{margin-bottom:10px}.results-block{background-color:#0072c6;color:#fff;margin-bottom:32px;padding:16px}.u-nomargin{margin:0}.u-margintop5{margin-top:5px}.local-header--question{padding:0}
```
Comment (st3v3nhunt) [inline error-503.html]: There is a warning due to the use of `Unicode Private Use Area`
screen shot 2017-12-04 at 15 54 16

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
A few changes based on an html validation run through.

File: error-503.html (hunk 1 above)
Comment (ajthomascouk) [reply error-503.html]: Good point. Spoke the Emma and she says unless we can determine which application the user is using when they got the error, we should not show the breadcrumb at all.

File: error-503.html (hunk 2 above)
Comment (ajthomascouk) [reply error-503.html]: Nope, not if the element is `<main>` well spotted.

File: error-503.html (hunk 9 above)
Comment (ajthomascouk) [reply error-503.html]: Slipped through the cracks. Removed.

File: error-503.html (hunk 6 above)
Comment (ajthomascouk) [reply error-503.html:35]: Added

File: error-503.html (hunk 5 above)
Comment (ajthomascouk) [reply error-503.html]: removed

File: error-503.html (hunk 2 above)
Comment (ajthomascouk) [reply error-503.html]: Removed

File: error-503.html (hunk 7 above)
Comment (ajthomascouk) [reply error-503.html]: Removed

File: error-503.html (hunk 8 above)
Comment (ajthomascouk) [reply error-503.html:96]: Added

File: error-503.html (hunk 3 above)
Comment (ajthomascouk) [reply error-503.html:107]: Formatting was well off. Sorted now

File: error-503.html (hunk 4 above)
Comment (ajthomascouk) [reply error-503.html]: Removed.

**Review by st3v3nhunt — CHANGES_REQUESTED** [review]
There are still a number of errors being highlighted by the W3 HTML check https://validator.w3.org/#validate_by_input
Please fix those which can be fixed or provide a reason as to why they aren't really a problem. Thanks!

Ideally this check should be added to the build pipeline so it happens automatically and doesn't require a manual check.

**Review by st3v3nhunt — APPROVED** [review]
Looking lovely 💖

**Pull request merged**